### ISSUE TITLE
Nodes don't respect "Watched" settings

### ISSUE THREAD

**Issue by gbooker** [croneter/PlexKodiConnect#823]
## Help yourself
- [X] I did try to restart Kodi :-)
- [X] I checked the [PKC Frequently Asked Questions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/faq)
- [X] I did try to reset the Kodi database by going to `PKC Settings -> Advanced -> "Reset the database and optionally reset PlexKodiConnect"` and then hitting YES, NO
- [X] I did check the [existing issues on Github](https://github.com/croneter/PlexKodiConnect/issues)

## Describe the bug
The settings in `Appearance Tweaks` concerning the presence of watched items in the recently added hubs are not respected.

## To Reproduce
1. Go to Settings -> `Appearance Tweaks`
2. Toggle the settings to include watched episodes or movies in the recently added hub
3. View the recently added hub for any sections
4. See watched items show up regardless of the setting

## Expected behavior
When the setting is not set to include watched items, then watched items would be excluded from the recently added hubs.

## You need to attach a KODI LOG FILE!
Logs not appropriate in this case.

## My Analysis
I searched the code on GH and discovered these settings are not reference in the code anywhere.  It appears this was lost in 5b7595c68126b55d66ba63df5f565c0cefd85a8e and after this change, recently added hubs always contain the same XML and do not include any filter for watched items.

I was able to get my desired behavior by adding this to the node XMLs manually:
```xml
  <rule field="playcount" operator="is">
    <value>0</value>
  </rule>
```

It's worth noting that other settings under `Appearance Tweaks` appear to also not be referenced in the code anywhere.  Perhaps they should be removed or re-implemented.

**Comment by croneter** [comment]
Should be solved in PKC **BETA** version 2.7.13. Please try and let me know here whether it worked!

**Comment by gbooker** [comment]
Appears to work.  I turned the settings back on, hit the `Reload Kodi node files to apply all the settings below`, restarted Kodi, and the unwatched showed up again in the recently added.  Repeated with turning them off and the unwatched disappeared.

The only issue is that I have to manually restart Kodi to actually see the difference.  Maybe the skin (Aeon Nox Silvo on Kodi 18.1) isn't refreshing the nodes when it should or something.
